plugins/GSdx/GSDevice11.h

```diff
@@ -58,6 +58,7 @@ class GSDevice11 : public GSDeviceDX
 	bool isNative;
 
 	bool UserHacks_unscale_pt_ln;
+	bool UserHacks_disable_NV_hack;
```

Comment:
I'm not sure how this style of not using ``m_`` prefix for member variables got introduced, but I'm merging this for now as this style seems to be present for most of the variables on this class.